### PR TITLE
bump prometheus with zipkin

### PR DESCRIPTION
**Pull request by zouyee**
Signed-off-by: zounengren <zounengren@cmss.chinamobile.com>

### 1. Why is this pull request needed and what does it do?
bump prometheus with zipkin

### 2. Which issues (if any) are related?
xref https://github.com/coredns/coredns/pull/4071

### 3. Which documentation changes (if any) need to be made?
none

### 4. Does this introduce a backward incompatible change or deprecation?
none

### CODE REVIEW

File: plugin/trace/README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ trace tracinghost:9253
 
 or
 
-~~~ corefile
```
Comment (miekg) [inline plugin/trace/README.md]: why this change?

Comment (zouyee) [reply plugin/trace/README.md]:
```
readme_test.go:81: Failed to start server with ../plugin/trace/README.md, for input "lookup : no such host":
        . {
            trace zipkin tracinghost:9253
        }
```

Comment (zouyee) [reply plugin/trace/README.md]: https://github.com/openzipkin/zipkin-go/blob/62dc8b26c05e0e8b88eb7536eff92498e65bbfc3/endpoint.go#L55

Comment (miekg) [reply plugin/trace/README.md]: can we make it localhost?

**Comment by zouyee** [comment]
/hold
The PR needs to pass the local test.

File: plugin/trace/README.md (hunk 1 above)
Comment (zouyee) [reply plugin/trace/README.md]: done

**Comment by zouyee** [comment]
@stp-ip PTAL

**Review by yongtang — APPROVED** [review]
LGTM
